extensions/amp-carousel/0.2/carousel.js

```diff
@@ -569,7 +577,7 @@ export class Carousel {
    * @private
    */
   resetSlideTransforms_() {
-    // TODO(sparhami) implement
+    this.slides_.forEach(slide => this.setElementTransform_(slide, 0, 0));
```

Comment:
nit: would be nice to annotate what `0, 0` are.

Reply:
I changed the code to be more clear here.